cli/cmd/buildhistory/buildhistory.go

```diff
@@ -30,6 +32,10 @@ func (b *BuildHistory) Customize(cmd *cli.Command) {
 }
 
 func (b BuildHistory) Run(ctx *clicontext.CLIContext) error {
+	if err := build.EnableBuildAndWait(ctx); err != nil {
+		logrus.Warn(err)
```

Comment:
Have been running into a bug where if user doesn't have access to that configmap in rio-system it will give an error. So instead I am just raising an error in here.